Source/engine/load_pcx.hpp

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <cstdint>
+
+#include <SDL.h>
+
+#include "engine/pcx_sprite.hpp"
+#include "utils/stdcompat/optional.hpp"
+
+namespace devilution {
+
+std::optional<OwnedPcxSprite> LoadPcxAsset(const char *path, std::optional<uint8_t> transparentColor = std::nullopt, SDL_Color *outPalette = nullptr);

```

Comment:
`LoadCel` isn't called `LoadCelAsset()`, isn't this inconstant with naming this function `LoadPcxAsset`?

Reply:
True, though I'd rather rename `LoadCel` to `LoadCelAsset`, to indicate that it's loaded via our assets mechanism.